src/app/core/services/parser.service.ts

```diff
@@ -0,0 +1,41 @@
+import { Injectable } from '@angular/core';
+
+/**
+ * Parser Service used for parsing strings and retrieving keywords
+ */
+@Injectable({
+  providedIn: 'root'
+})
+export class ParserService {
```

Comment:
We don't need to create a service here. The `extractStringBetween` function can just be exported from a module under `src/app/shared/lib`

Reply:
Remember to make this change :+1:  @zatkiller

Reply:
Will fix this in the new commits 😄